collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java

```diff
@@ -105,45 +104,8 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-
```

Comment:
It seems that the code is moving from up here to down below. Is there a reason to do this?